language/tools/move-unit-test/src/test_reporter.rs

```diff
@@ -231,7 +231,12 @@ impl TestFailure {
                             .source_map
                             .get_function_source_map(*fdef_idx)
                             .ok()?;
-                        let loc = function_source_map.get_code_location(*offset)?;
+                        // TODO: the code offset from the VMError should be an option type so we
+                        // don't have to look for it and then default to the function declaration
+                        // location here for native functions.
```

Comment:
I'm not sure I understand the TODO here. Could you paste what the VMError/Status is we get from this function? It would help me a lot

Reply:
We get the following `VMError` from an abort in the `Vector::borrow` function in the tests:
```rust
VMError {
    major_status: ABORTED,
    sub_status: Some(
        1,
    ),
    message: None,
    location: Module(
        ModuleId {
            address: 00000000000000000000000000000001,
            name: Identifier(
                "Vector",
            ),
        },
    ),
    indices: [],
    offsets: [
        (
            FunctionDefinitionIndex(1),
            0, // <- This is the issue, not bytecode offset of 0 since it's a native function
        ),
    ],
}
```

Reply:
ah gotcha. I thought it was the function def as an issue. 
We might be stuck with this being zero though :/ Do you think it would be an issue long term?